server/auth/github/client.go

```diff
@@ -173,6 +177,29 @@ func (c *Client) IsMember(organization, token string) (bool, error) {
 	return true, nil
 }
 
+// IsTeamMember returns true if the given user is a member of the team.
+func (c *Client) IsTeamMember(teamId, name string, token string) (bool, error) {
```

Comment:
ok got it. just dug through how auth works for a request and I see now that we only call `authenticate` via github the first time.